worker.sh

```diff
@@ -9,7 +9,7 @@ ROOTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd ${ROOTDIR}
 
 # Get vars from yaml
-QUEUE=$(cat ${ROOTDIR}/instance_config.yaml | grep celery_worker_queue)
+QUEUE=$(cat ${ROOTDIR}/static_config.yaml | grep celery_worker_queue)
```

Comment:
@muhammad-ammar celery worker queues are more than one. Will this work?

Reply:
As discussed with @yro this should work. @yro what do you say?